src/TransCreator/index.jsx

```diff
@@ -1,10 +1,68 @@
-import React from 'react';
-import { View, Text } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import {
+  Container,
+  Content,
+  Item,
+  Root,
+} from 'native-base';
+import {} from 'react-native';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { NavigationScreenPropType } from 'react-navigation';
+import styles from './Style';
+import PageHeader from './PageHeader';
+import PageFooter from './PageFooter';
+import AmountInput from './AmountInput';
+import DateSelector from './DateSelector';
+import { setNewTransactionInsertionSuccess } from '../Stats/actionCreator';
+
+const TransCreator = ({ navigation }) => {
+  const [transAmount, setTransAmount] = useState(null);
+  const [transType, setTransType] = useState('Expense');
+  const [transDate, setTransDate] = useState(moment().unix());
+
+  const { newTransInsertionSuccess } = useSelector((state) => state.Transactions);
```

Comment:
- you are using the wrong state value.
- also i am not sure if we need this newTransInsertionSuccess, you can check the length of the transaction as the condition for the useEffect

Reply:
fixed